src/sqlfluff/dialects/postgres_keywords.py

```diff
@@ -0,0 +1,899 @@
+    ("ASC", "reserved"),
+    ("ASENSITIVE", "not-keyword"),
+    ("ASIN", "not-keyword"),
+    ("ASSERTION", "non-reserved"),
+    ("ASSIGNMENT", "non-reserved"),
+    ("ASYMMETRIC", "reserved"),
+    ("AT", "non-reserved"),
+    ("ATAN", "not-keyword"),
+    ("ATOMIC", "not-keyword"),
+    ("ATTACH", "non-reserved"),
+    ("ATTRIBUTE", "non-reserved"),
+    ("ATTRIBUTES", "not-keyword"),
+    ("AUTHORIZATION", "reserved-(can-be-function-or-type)"),
+    ("AVG", "not-keyword"),
+    ("BACKWARD", "non-reserved"),
+    ("BASE64", "not-keyword"),
+    ("BEFORE", "non-reserved"),
+    ("BEGIN", "non-reserved"),
+    ("BEGIN_FRAME", "not-keyword"),
+    ("BEGIN_PARTITION", "not-keyword"),
+    ("BERNOULLI", "not-keyword"),
+    ("BETWEEN", "non-reserved-(cannot-be-function-or-type)"),
+    ("BIGINT", "non-reserved-(cannot-be-function-or-type)"),
+    ("BINARY", "reserved-(can-be-function-or-type)"),
+    ("BIT", "non-reserved-(cannot-be-function-or-type)"),
+    ("BIT_LENGTH", "not-keyword"),
+    ("BLOB", "not-keyword"),
+    ("BLOCKED", "not-keyword"),
+    ("BOM", "not-keyword"),
+    ("BOOLEAN", "non-reserved-(cannot-be-function-or-type)"),
+    ("BOTH", "reserved"),
+    ("BREADTH", "not-keyword"),
+    ("BY", "non-reserved"),
+    ("C", "not-keyword"),
+    ("CACHE", "non-reserved"),
+    ("CALL", "non-reserved"),
+    ("CALLED", "non-reserved"),
+    ("CARDINALITY", "not-keyword"),
+    ("CASCADE", "non-reserved"),
+    ("CASCADED", "non-reserved"),
+    ("CASE", "reserved"),
+    ("CAST", "reserved"),
+    ("CATALOG", "non-reserved"),
+    ("CATALOG_NAME", "not-keyword"),
+    ("CEIL", "not-keyword"),
+    ("CEILING", "not-keyword"),
+    ("CHAIN", "non-reserved"),
+    ("CHAINING", "not-keyword"),
+    ("CHAR", "non-reserved-(cannot-be-function-or-type)"),
+    ("CHARACTER", "non-reserved-(cannot-be-function-or-type)"),
+    ("CHARACTERISTICS", "non-reserved"),
+    ("CHARACTERS", "not-keyword"),
+    ("CHARACTER_LENGTH", "not-keyword"),
+    ("CHARACTER_SET_CATALOG", "not-keyword"),
+    ("CHARACTER_SET_NAME", "not-keyword"),
+    ("CHARACTER_SET_SCHEMA", "not-keyword"),
+    ("CHAR_LENGTH", "not-keyword"),
+    ("CHECK", "reserved"),
+    ("CHECKPOINT", "non-reserved"),
+    ("CLASS", "non-reserved"),
+    ("CLASSIFIER", "not-keyword"),
+    ("CLASS_ORIGIN", "not-keyword"),
+    ("CLOB", "not-keyword"),
+    ("CLOSE", "non-reserved"),
+    ("CLUSTER", "non-reserved"),
+    ("COALESCE", "non-reserved-(cannot-be-function-or-type)"),
+    ("COBOL", "not-keyword"),
+    ("COLLATE", "reserved"),
+    ("COLLATION", "reserved-(can-be-function-or-type)"),
+    ("COLLATION_CATALOG", "not-keyword"),
+    ("COLLATION_NAME", "not-keyword"),
+    ("COLLATION_SCHEMA", "not-keyword"),
+    ("COLLECT", "not-keyword"),
+    ("COLUMN", "reserved"),
+    ("COLUMNS", "non-reserved"),
+    ("COLUMN_NAME", "not-keyword"),
+    ("COMMAND_FUNCTION", "not-keyword"),
+    ("COMMAND_FUNCTION_CODE", "not-keyword"),
+    ("COMMENT", "non-reserved"),
```

Comment:
I was testing this branch today, since it has a LOT of the keywords that are currently unparseable in my postgresql files - thanks! I found one example that triggered a parsing failure, and wondered how/best to add to this changeset.

This SQL

```sql
COMMENT ON FUNCTION dummy() IS NULL;
```

raises:

```
sqlfluff.api.simple.APIParsingError: Found 1 issues while parsing string.
Line 2, Position 1: Found unparsable section: 'COMMENT ON FUNCTION dummy() IS NULL'
```

Reply:
That won't be a keyword problem, it's probably undeveloped grammar, can you add this as an issue?

Reply:
Done!